packages/apsara-ui/src/Listing/Listing.types.tsx

```diff
@@ -1,28 +1,26 @@
 import { IVirtualTable } from "../Table/VirtualisedTable";
-export interface ListingProps {
-    list?: any[];
+import { Column } from "../TableV2/VirtualisedTable";
+export interface ListingProps<T> {
+    list?: T[];
     loading?: boolean;
     resourceName?: string;
     resourcePath?: string;
     rowKey?: string;
     className?: string;
     tableProps?: {
-        getColumnList?: any;
-        handleRowClick?: (event: any, rowIndexData: any) => void;
+        getColumnList?: (path: string, sortedInfo: any) => Column<T>[];
```

Comment:
Let's create a type for `getColumnList`.
example

```typescript
export type ColumnRenderFunc = (path: string, sortedInfo: any) => Column<T>[];

and change to
getColumnList?: ColumnRenderFunc;
```

This will help when users are defining function in different file.